app/controllers/submissions_controller.rb

```diff
@@ -455,13 +467,12 @@ def update_files
             m_logger.log(msg)
           end
         else
-          @file_manager_errors[:update_conflicts] = txn.conflicts
+          flash_message(:error, partial: 'submissions/file_conflicts_list')
```

Comment:
This partial seems to use the removed `@file_manager_errors`. You could fix this by passing `txn.conflicts` as a [local variable](http://guides.rubyonrails.org/layouts_and_rendering.html#local-variables) to the partial.